### DISCUSSION TITLE
Custom dictionary list for autocomplete

### DISCUSSION THREAD

**Discussion by hrai** [helix-editor/helix#4376] [Ideas]
I have been using coc.nvim and was wondering if it's possible to have something similar to [coc-dictionary](https://github.com/neoclide/coc-sources).

There's also a feature where words from current open buffers is used in autocomplete.

This would significantly reduce typing content and would be the biggest driver for me to move to helix which I'm already loving after using it for a few days.

**Comment by CptPotato** [comment]
This is currently not possible to my knowledge, unless there is a language server that provides this functionality (since auto-completion is based on the language server for now).

There's a related issue #1063
